tests/routes/metadata_test.py

```diff
@@ -39,6 +39,15 @@ def test_put_metadata_invalid_json(self):
             .put(data='{"lol": ...}', headers=self.auth)
 
     def test_empty_metadata(self):
+        """
+            This will ensure that things are initialized
+            correctly.
+
+            See pyshelf.metadata.initializer.Initializer
+
+            For the "empty" artifact see
+            tests.functional_test_base.FunctionalTestBase.setup_artifacts
+        """
```

Comment:
Thanks for adding this. I should have documented this. I remember doing this now.